integration-tests/general/tests/src/test/kotlin/com/r3/conclave/integrationtests/general/tests/filesystem/EnclaveRestartFileSystemTest.kt

```diff
@@ -15,6 +16,7 @@ class EnclaveRestartFileSystemTest : FileSystemEnclaveTest() {
 
     @Test
     fun `cannot read previously saved files after enclave restarts when using in-memory filesystem`() {
+        graalvmOnlyTest() // CON-1264: Gramine: accessing filesystem and devices causes InvalidKeyException: Invalid AES key length: 0 bytes
```

Comment:
Shouldn't all the file system tests (beside the /tmp ones) reference CON-1259 (i.e. throw the error for 1.4)?

Reply:
I was thinking about it, but they don't fail because of that. It looks like a more generic error at the moment and whoever picks CON-1264 would know what tests are failing with that error.